receiver/prometheusremotewritereceiver/receiver.go

```diff
@@ -232,16 +250,42 @@ func (prw *prometheusRemoteWriteReceiver) translateV2(_ context.Context, req *wr
 			continue
 		}
 
+		// If the metric name is equal to target_info, we use its labels as attributes of the resource
+		// Ref: https://opentelemetry.io/docs/specs/otel/compatibility/prometheus_and_openmetrics/#resource-attributes-1
+		if ls.Has(labels.MetricName) && ls.Get(labels.MetricName) == "target_info" {
+			var rm pmetric.ResourceMetrics
+			hashedLabels := xxhash.Sum64String(ls.Get("job") + string([]byte{'\xff'}) + ls.Get("instance"))
+
+			if existingRM, ok := prw.interRequestCache[hashedLabels]; ok {
+				rm = existingRM
+			} else {
+				rm = otelMetrics.ResourceMetrics().AppendEmpty()
+			}
+
+			attrs := rm.Resource().Attributes()
+			parseJobAndInstance(attrs, ls.Get("job"), ls.Get("instance"))
+
+			// Add the remaining labels as resource attributes
+			for _, l := range ls {
+				if l.Name != "job" && l.Name != "instance" && l.Name != labels.MetricName {
+					attrKey := strings.ReplaceAll(l.Name, "_", ".")
```

Comment:
```suggestion
```

We do this translation when sending OTLP -> Prometheus, but this requirement doesn't exist for the other way around

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/38812/commits/d2ae2f440cde588a3a01084b12a9e80ef2bb28d5